website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,129 @@
+Karpenter has the ability to efficiently address the full range of instance types available through AWS.
+
+* **Adds scheduling**: Cluster autoscaler doesn’t bind pods to the nodes it creates.
+A node that Karpenter creates knows to run the bound pods immediately when the node comes up.
+I doesn't have to wait for the scheduler or for the node to become ready.
+It can start pulling the container images it needs immediately.
+This can save seconds of performance.
+
+## Constraints
+Understanding the concept of layered constraints is key to using Karpenter.
+With no constraints defined in provisioners and none requested from pods being deployed, Karpenter on AWS is free to choose from the entire universe of features available from EC2.
+Nodes can be created using any instance type and run in any zones.
+On-demand or spot instances can be used.
+
+The effects of layering constraints are applied when a pod makes requests for compute resources that cannot be met by current capacity.
+A pod can request *affinity* (such as to run in a particular zone or instance type) or a topology spread (to require that a set of pods be run across multiple nodes).
+The pod can ask to run only on nodes with a particular label or that have a certain amount of available memory or particular architecture type.
+
+The Kubernetes scheduler then tries to match those constraints with available nodes.
+If the node is unschedulable, facilities like the cluster autoscaler or Karpenter come in and try to create compute resources to match each pod's needs.
+When Karpenter tries to provision a node that can schedule an unschedulable pode, it compares its own constraints when choosing the node to create.
+The Karpenter provisoners configured for the cluster will look to best match the request, using mostly the same well-known labels and affinity features that the pod uses.
+
+For details about how to set constraints when configuring a Karpenter provider, see [Karpenter provisioner and scheduler](/docs/concepts/provisioner.md) for details.
+For more on how, as a developer, you can add constraints to your pod spec deployment, see [Karpenter pod developer concepts](/docs/concepts/running-pods.md) for details.
+
+## Deprovisioning nodes
+
+Karpenter not only manages the creation of nodes to add cluster capacity, but also will delete nodes when that capacity is no longer needed.
+There are several things you should know about how Karpenter deprovisions nodes:
+
+* **Node Expiry**: If a node expiry time-to-live value (`ttlSecondsUntilExpired`) is reached, that node is drained of pods and deleted (even if it is still running workloads).
+* **Empty nodes**: When the last workload pod running on a Karpenter-managed node is gone, a clock starts keeping track of how long the node is empty.
+Once that "node empty" time-to-live (`ttlSecondsAfterEmpty`) is reached, that node is deleted.
+* **Finalizer**: Karpenter places a finalizer bit on the nodes it creates.
```

Comment:
Thoughts putting this first and explaining cordon/drain once, since it's shared between all flows.